packages/salesforcedx-vscode-apex-debugger/package.json

```diff
@@ -53,7 +53,11 @@
       "shx rm -rf node_modules && shx rm -rf out && shx rm -rf coverage && shx rm -rf .nyc_output",
     "postinstall": "node ./node_modules/vscode/bin/install",
     "test": "node ../../scripts/run-test-with-top-level-extensions",
-    "test:unit": "node ../../scripts/run-test-with-top-level-extensions"
+    "test:insiders":
```

Comment:
Both `test:insiders` and `test:unit-insiders` run in the same way, why do we need these 2 scripts ? I know you are following the same structure we currently have (`test` & `test:unit`) but I want to make sure we only add the scripts we need.

Reply:
So looking at all of the package.jsons it looks like the `test` and `test:unit` scripts are the exact same for all of them. It seems as though the unit tests are the exact same as the others, so I'm not sure that we need the `test:unit` or `test:unit-insiders` scripts.

Reply:
Ok, for now let's just focus on removing `test:unit-insiders` and log a work item to get rid of `test:unit` in the future.

Reply:
Sounds good, I'll create a work item for it as well.